sklearn/covariance/robust_covariance.py

```diff
@@ -361,11 +361,7 @@ def fast_mcd(X, support_fraction=None,
     """
     random_state = check_random_state(random_state)
 
-    X = np.asarray(X)
-    if X.ndim == 1:
-        X = np.reshape(X, (1, -1))
-        warnings.warn("Only one sample available. "
-                      "You may want to reshape your data array")
+    X = check_array(X, ensure_min_samples=2, estimator='MinCovDet')
```

Comment:
Better use `estimator='fast_mcd'` even if this is a slight abuse of notation, it is more precise for the resulting error message.